### PR TITLE
Fix Style/AlignArray

### PR DESCRIPTION
**Pull request by esasse**

### CODE REVIEW

**Comment by indirect** [comment]
@homu r+

File: spec/bundler/gem_helper_spec.rb [hunk 1]
```diff
@@ -104,7 +104,7 @@ def remove_push_guard(gemspec_content)
 
       context "defines Rake tasks" do
         let(:task_names) { %w[build install release
-          release:guard_clean release:source_control_push release:rubygem_push] }
+                              release:guard_clean release:source_control_push release:rubygem_push] }
```
Comment (segiddins) [inline spec/bundler/gem_helper_spec.rb]: this could use a third line

Comment (esasse) [inline spec/bundler/gem_helper_spec.rb]: done!

**Comment by segiddins** [comment]
@homu r+

**Comment by esasse** [comment]
Maybe a connection problem?

**Pull request merged**